### ISSUE TITLE
AWS Swapping Revision Number on Release

### ISSUE THREAD

**Issue by ColinBuyck** [metrotranscom/doorway#580]
A recent release broke email functionality in Doorway briefly because it unexpectadly used a very old set of environment variables (revision number 4). This ticket is to investigate why that occurs and how to prevent releases from taking old revision sets.

**Comment by sarahlazarich** [comment]
@ColinBuyck is this still relevant?

**Comment by ColinBuyck** [comment]
Yes. We've sort of resolved it by deleting all other revision version numbers but that's not the most sustainable approach.

**Comment by sarahlazarich** [comment]
hasn't bitten us recently, but still need a long term fix
